README.md

```diff
@@ -4,9 +4,21 @@ Java API for BBj DWC Dynamic Web Client
 
 ## Setup Instructions
 
-1. Download and Install BBj 22.12 or higher, on OpenJDK11
-2. Clone this project
-3. Add a custom classpath with
+1. Download and Install BBj 22.13 or higher, on OpenJDK17
+
+2. Installation of needed deps
+To install the bbj lib dependencies locally, execute the following three lines in your
+local `<bbx>/lib directory`:
+
+```shell
```

Comment:
In windows, this will fail. quotes should be added for every parameter

Reply:
This is something a "dwcj cli" can do

Reply:
Currently broken anyway, in maven 3.9 at least
https://issues.apache.org/jira/browse/MNG-7679

However, I didn't see an issue on Windows with the params, did you run into any?

Reply:
That did not work for me with maven `3.8.6`. I had to add the quotes 

```ssh
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.068 s
[INFO] Finished at: 2023-02-27T13:13:12+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] The goal you specified requires a project to execute but there is no POM in this directory (C:\Users\habof). Please verify you invoked Maven from the correct directory. -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MissingProjectException
```

Reply:
what would have been the command line for Windows?

Reply:
1. `mvn install:install-file "-Dfile=C:/bbj/lib/BBjStartup.jar" "-DgroupId=com.basis.lib" "-DartifactId=BBjStartup" "-Dversion=1.0" "-Dpackaging=jar"`
2. `mvn install:install-file "-Dfile=C:/bbj/lib/BBj.jar" "-DgroupId=com.basis.lib" "-DartifactId=BBj" "-Dversion=1.0" "-Dpackaging=jar"`
3. `mvn install:install-file "-Dfile=C:/bbj/lib/BBjUtil.jar" "-DgroupId=com.basis.lib" "-DartifactId=BBjUtil" "-Dversion=1.0" "-Dpackaging=jar"`